azure-pipelines.yml

```diff
@@ -38,41 +38,49 @@ stages:
         echo "##vso[task.setvariable variable=docker_path;]$docker_path"
       displayName: 'Set version number'
 
-    - script: |
-        make setup
+    - script: make setup
       displayName: 'Build & setup'
       env:
         dockerHubUsername: $(dockerHubUsername)
         dockerHubImageName: $(imageName)
-        GOVUK_NOTIFY_API_KEY: $(govukNotifyAPIKey)
+        railsSecretKeyBase: $(railsSecretKeyBase)
+        RAILS_ENV: production
```

Comment:
How do we create the test db here if RAILS_ENV is `production`? Going to have a look at this locally

Reply:
There is a step in the azure pipeline to run the db:setup again in the container that used during build.